src/tests/integration/mapping.test.ts

```diff
@@ -42,6 +43,30 @@ describe('/api', () => {
       })
   })
 
+  it('Delete mapping', async () => {
+    const subDomain = `delete${uuidv4()}`
+    const domain = 'albertow'
+    const port = '4500'
+    const createMapping = await mappingAdapter('/', 'POST', {
+      domain,
+      subDomain,
+      port,
+    })
+    expect(createMapping.status).toEqual(200)
+    const mapping = await createMapping.json()
+    const delMapping = await mappingAdapter(`/delete/${mapping.id}`, 'DELETE')
+    expect(delMapping.status).toEqual(200)
+    const deletedMapping = await delMapping.json()
+    const getDeletion = await mappingAdapter(`/${mapping.id}`, 'GET')
```

Comment:
`getMapping` -> This API is to get a mapping. getting deletion doesn't make sense.